assets/state/state.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+{{.path}} "$@"
```

Comment:
```suggestion
"{{.path}}" "$@"
```

I think we'd need this on macOS/Linux as well, for paths with spaces (however unlikely that might be on those platforms).